tests/helpers/mock-parse.js

```diff
@@ -0,0 +1,3 @@
+export default function mockParse() {
```

Comment:
FWIW, we usually do this kind of global mock in a single file. For example:

* https://github.com/aptible/ember-cli-aptible-shared/blob/master/test-support/helpers/mock-stripe.js#L3

And usage for tests:

* https://github.com/aptible/ember-cli-aptible-shared/blob/master/tests/test-helper.js#L20